src/logic/calculate.js

```diff
@@ -0,0 +1,16 @@
+import Operate from './operate';
+
+const Calculate = (() => {
+  const calculate = (dataObject, buttonName) => {
+    // eslint-disable-next-line
+    var { total, next, operation } = dataObject;
+    if (buttonName === operation) {
+      Operate.operate(total, next, operation);
+    }
```

Comment:
- [ ] We need to evaluate more cases here. What should happen if the user press `AC`, `.` or `+/-`? We also need to address these cases.